ios/FluentUI.Demo/FluentUI.Demo/Demos/TooltipDemoController.swift

```diff
@@ -103,11 +103,7 @@ class TooltipDemoController: DemoController {
             Tooltip.shared.show(with: "This is a very long, offset message.", for: sender.view, preferredArrowDirection: .left, screenMargins: margins)
         }).view
 
-        let bottomRightButton = createButton(title: "", action: { [weak self] sender in
-            guard let strongSelf = self else {
-                return
-            }
-
+        let bottomRightButton = createButton(title: "", action: { sender in
```

Comment:
Why was the reference cycle prevention removed?

Reply:
That is just because there's no use of self in the implementation of the closure to justify its capture.